market/uipath/meta.json

```diff
@@ -20,16 +31,26 @@
 			"data": {
 				"openApiUrl": "https://platform.uipath.com/AXONPRESALES/AXONPRESALES/swagger/v11.0/swagger.json",
 				"namespace": "com.uipath.orchestrator",
+				"clientId": "699e715f-63b1-4355-a974-ee3cac26985e", 
+				"baseUri": "https://platform.uipath.com/",
 				"features": [
-					"com.axonivy.connectivity.rest.client.auth.OAuth2Feature",
+					"com.uipath.orchestrator.feature.OAuth2Feature",
 					"com.uipath.orchestrator.feature.TenantHeaderFeature",
 					"ch.ivyteam.ivy.rest.client.mapper.JsonFeature"
 				],
 				"properties": [
 					{
-						"name": "tenant",
+						"name": "AUTH.tenant",
 						"value": "AXONPRESALES"
 					},
+					{
+						"name": "AUTH.clientId",
+						"value": ""
```

Comment:
the rest client installer could be more intelligent. if "value" is not set,then just create the property with empty value. but good for now

Reply:
yes, good idea. for now I think having empty values helps the user a bit to customize the installer ... since the valid structure to fill in a real value is already present and able to consume custom values at install time :+1:  ... but let's see whether we stick to that kind of customization in the future